core/theme/ThemeManager.php

```diff
@@ -234,4 +234,12 @@ protected function setActiveTheme(Theme $theme)
     {
         $this->_activeTheme = $theme;
     }
+    
+    /**
+     * @return bool
+     */
+    public function hasActiveTheme()
```

Comment:
if you rename it to `getHasActiveTheme()` its possible to retrieve as `hasActiveTeheme` property.